widget/radio_test.go

```diff
@@ -292,6 +294,22 @@ func TestRadio_Hovered(t *testing.T) {
 			assert.Equal(t, 1, radio.hoveredItemIndex)
 			assert.Equal(t, theme.BackgroundColor(), render.items[0].focusIndicator.FillColor)
 			assert.Equal(t, theme.HoverColor(), render.items[1].focusIndicator.FillColor)
+
+			// Focus indicators should be centered vertically and aligned left horizontally
```

Comment:
Where a comment is inserted it usually means the meaning is not clear in the code. In this instance it would be more clear if it was in a new test with a descriptive name

Reply:
Done, I moved the test to separate method, pls take a look again.